src/builder/create_sticker.rs

```diff
@@ -15,13 +15,14 @@ use crate::model::prelude::*;
 /// - [`PartialGuild::create_sticker`]
 /// - [`Guild::create_sticker`]
 /// - [`GuildId::create_sticker`]
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
 #[must_use]
 pub struct CreateSticker<'a> {
     name: String,
     tags: String,
     description: String,
     file: CreateAttachment,
+    #[serde(skip)]
     audit_log_reason: Option<&'a str>,
 }
```

Comment:
```suggestion
#[derive(Clone, Debug, Serialize, Deserialize)]
#[must_use]
pub struct CreateSticker<'a> {
    name: String,
    tags: String,
    description: String,
    
    #[serde(skip)]
    file: CreateAttachment,
    #[serde(skip)]
    audit_log_reason: Option<&'a str>,
}
```
I think the `file` field should also be skipped for consistency.